recipes/folly/all/conanfile.py

```diff
@@ -144,6 +150,9 @@ def source(self):
     def _configure_cmake(self):
         cmake = CMake(self)
         if can_run(self):
+            if self.options.with_sse4_2:
```

Comment:
Since when is it available? Is it backward compatible?

Reply:
> 

sse4.2 support was in folly long long ago (>7 years)
I will set option default to False to be backward compatible.